Dockerfile

```diff
@@ -0,0 +1,142 @@
+################################
+# Python dependencies builder
+#
+FROM python:2-stretch AS base
+
+WORKDIR /app
+EXPOSE 8000
+
+ENV LANG=C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PATH="/venv/bin:$PATH"
+
+RUN virtualenv /venv
+RUN useradd -d /app -M --uid 1000 --shell /usr/sbin/nologin kitsune
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+            gettext build-essential \
+            libxml2-dev libxslt1-dev zlib1g-dev git \
+            libjpeg-dev libffi-dev libssl-dev libxslt1.1 \
+            libmariadbclient-dev mariadb-client && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY ./requirements/*.txt /app/requirements/
+
+RUN pip install --no-cache-dir --require-hashes -r requirements/default.txt && \
+    pip install --no-cache-dir --require-hashes -r requirements/dev.txt && \
+    pip install --no-cache-dir --require-hashes -r requirements/test.txt
+
+ARG GIT_SHA=head
```

Comment:
I think the `ARG` should be at begining of the file

Reply:
my understanding is that it just needs to come before you use it. This is mostly copied straight from the split up dockerfiles.